src/vs/workbench/services/runtimeSession/common/runtimeSession.ts

```diff
@@ -452,6 +471,14 @@ export class RuntimeSessionService extends Disposable implements IRuntimeSession
 			startMode,
 			true
 		);
+
+		if (notebookUri) {
+			// Make sure that the notebook knows about its session id for proper mapping of notebook file name to session.
+			await this._commandService.executeCommand('_positron.storeNotebookSessionId', notebookUri.toString(), sessionName)
```

Comment:
The `_positron.storeNotebookSessionId` command doesn't seem to be defined anywhere, which also makes me wonder if we need the `sessionName` changes above?

Reply:
No need for the command but the session name changes are important so we dont have problems when the user opens up two untitled notebooks within a single positron sessions as then both would have the same ID as the session id doesn't change when we rename, just the label as retrieved by `currentNotebookUri` in the dynamic metadata.

Reply:
Actually, no, you're right. We can totally get rid of the sessionName variable because notebook sessions no longer need to be checked when generating the id as it gets updated in the `generateNewSessionId()` method. Great catch!